test/distributed/_spmd/test_tracing.py

```diff
@@ -516,7 +516,9 @@ class AssertOverride(Override):
             def __init__(self, outer):
                 self.outer = outer
 
-            def replacement(self, orig_submodule: torch.nn.Module) -> torch.nn.Module:
+            def replacement(
+                self, orig_submodule: torch.nn.Module, fqn: str
```

Comment:
nit: should `fqn` be the first argument instead of `orig_submodule`?

Reply:
good point, let me update.